config.yml

```diff
@@ -3578,6 +3762,7 @@ nodes:
     fields:
       - name: body
         type: node[]
+        kind: Node # fun fact: the only place where absolutely every node is valid
```

Comment:
This comment isn't entirely correct. There are lots of inner nodes that wouldn't be valid here, like hash contents or parameters. I think we should probably remove this comment.

Reply:
OK, removed.